BUILDING.md

```diff
@@ -1,255 +1,49 @@
-# Building VTR #
+# Building VTR
 
 
-## Overview ##
+## Setting up Your Environment
 
-VTR uses [CMake](https://cmake.org) as it's build system.
 
-CMake provides a portable cross-platform build systems with many useful features.
+VTR requires several system packages.  From the top-level directory, run the following script to install the required packages on a modern Debian or Ubuntu system:
 
-## Tested Compilers ##
-VTR requires a C++-14 compliant compiler.
-It is tested against the default compilers of all Debian and Ubuntu releases within their standard support lifetime. Currently, those are the following:
+    ./install_apt_packages.sh
 
-* GCC/G++: 7, 8, 9, 10, 11
-* Clang/Clang++: 6, 7, 10
 
-Other compilers may work but are untested (your milage may vary).
+You will also need several Python packages.  You can optionally install and activate a Python virtual environment so that you do not need to modify your system Python installation:
 
-## Unix-like ##
-For unix-like systems we provide a wrapper Makefile which supports the traditional `make` and `make clean` commands, but calls CMake behind the scenes.
+    make env
+    source .venv/bin/activate
```

Comment:
Should the source .venv/bin/activate command be done by make env? (one less command to run).

Reply:
Unfortunately there's no way to affect the environment of the caller's process (the user's terminal) from within a Makefile command.  https://stackoverflow.com/a/7508273

What I've seen done in other projects (including several Symbiflow projects) is to activate the environment from within a Makefile target, which would have effect for only that target.  Since we don't run the VTR flow using Makefile targets (but rather have users directly invoke the Python scripts) that doesn't really help.

Reply:
OK, that's a good reason :).